### ISSUE TITLE
Feature request: support EPS files for image previews

### ISSUE THREAD

**Issue by darsnack** [notaapp/issues#57]
For eventual LaTeX output (common in academic publishing), image files as saved as EPS to allow resizing for the published PDF. Currently EPS files don't render in the preview, and they aren't recognized in the editor for thumbnails.

**Comment by astoilkov** [comment]
I saw that you are using iA Writer and Ulysses — do they support this?

I researched if this can be a quick thing we can add. However, there doesn't seem to be an easy way. I will need to research this more.

Thanks for the suggestion.

**Comment by darsnack** [comment]
Yes, both seem to support it. Not a high priority feature request for me. Just something I noticed when importing my notes.
